src/components/page/CustomCallback/tests/index.test.tsx

```diff
@@ -0,0 +1,164 @@
+import { mount } from 'enzyme';
```

Comment:
These tests results in some warnings in the terminal i.e. something like 

```
  console.error
    The above error occurred in the <BaseUnsuccessfulLogin> component:
        in BaseUnsuccessfulLogin (created by Context.Consumer)
        in withRouter(BaseUnsuccessfulLogin) (created by BaseCallbackComponent)
        in BaseCallbackComponent (created by APICallback)
        in APICallback (created by ConnectFunction)
        in ConnectFunction (at CustomCallback/index.tsx:79)
        in CustomConnectedAPICallBack (at tests/index.test.tsx:44)
        in Router (at tests/index.test.tsx:43)
        in Provider (created by WrapperComponent)
        in WrapperComponent

```

Can we get rid of the warnings?

Reply:
addressed via https://github.com/OpenSRP/web/pull/9/commits/d7116c44b1b4cd9a025d7d485fc840aff0a09f39